Document newlines in textareas (#438)
CloudCannon/platform-documentation: articles/text-inputs.mdx

<comp.CodeBlock language="Markdown" source="about.md">
```

---
blog_title: The benefits of static sites
---

Page content goes here.

```
</comp.CodeBlock>

## Textarea

> The Textarea input provides an editing interface for plain text which is resizable and allows newlines to be added.

Reviewer: ```suggestion
The Textarea input provides an editing interface for plain text. This area is resizable and allows you to add newlines.
```
